bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -1214,7 +1242,6 @@ protected function updateIndexesOfObjectSiblings(DataObject\AbstractObject $upda
             'o_parentId = ? AND o_id != ?',
             [$updatedObject->getParentId(), $updatedObject->getId()]
         );
-        $list->setObjectTypes([DataObject\AbstractObject::OBJECT_TYPE_OBJECT, DataObject\AbstractObject::OBJECT_TYPE_VARIANT, DataObject\AbstractObject::OBJECT_TYPE_FOLDER]);
```

Comment:
on purpose? 
seems you started from an old tag?

Reply:
fixed